RULES/xilinx/bitstream.go

```diff
@@ -115,7 +144,7 @@ func (rule Bitstream) Build(ctx core.Context) {
 	ins := []core.Path{}
 	for _, ip := range hdl.FlattenIpGraph(rule.Ips) {
 		for _, src := range ip.Sources() {
-			if strings.HasSuffix(src.String(), ".v") || strings.HasSuffix(src.String(), ".sv") {
+			if strings.HasSuffix(src.String(), ".v") || strings.HasSuffix(src.String(), ".sv") || strings.HasSuffix(src.String(), ".vhd") {
```

Comment:
Since this list appears in multiple places now: What about a map / slice with these extensions in a single place?

Reply:
Good idea.